### PR TITLE
feat: Added csdiff and git packages

### PR DESCRIPTION
**Pull request by jperezdealgaba**
csdiff and git packages are installed in order to be used by SAST tasks to parse the results and generate fingerprinting.

This will be used by future SAST tasks provided by the OpenScanHub team, for example: https://issues.redhat.com/browse/OSH-737

### CODE REVIEW

**Comment by jperezdealgaba** [comment]
@ralphbean Would you mind giving a review on this?

**Review by jsztuka — APPROVED** [review]
lgtm

**Comment by jperezdealgaba** [comment]
@ralphbean @14rcole @Josh-Everett Could any of you review it and approve it/comment it? Thank you!

**Review by ralphbean — APPROVED** [review]

**Comment by jperezdealgaba** [comment]
@ralphbean Would you mind enabling the last CI test? I am not able to trigger it/merge this

**Comment by dirgim** [comment]
@jperezdealgaba if you rebase your commit on the latest `main`, the Red Hat Konflux job issue should get resolved.

**Comment by jperezdealgaba** [comment]
@dirgim I rebased the PR and GitHub is still showing that I need the approval from one maintainer for the workflow:

Screenshot 2024-09-09 at 15 58 16

P.S.: I added the installation of the git packages to this PR as it will be also needed. I hope it is not a problem

**Comment by jperezdealgaba** [comment]
All checks passed! I have no merge rights in this repo

**Review by dirgim — APPROVED** [review]
+1

**Pull request merged**
